src/Avalonia.Base/AvaloniaObject.cs

```diff
@@ -599,9 +599,9 @@ private object GetDefaultValue(AvaloniaProperty property)
         private object GetValueOrDefaultUnchecked(AvaloniaProperty property)
         {
             var aobj = this;
-            if (aobj.Values != null)
+            if (aobj._values != null)
```

Comment:
Could just store this as local to avoid multiple field accesses.

Reply:
I will do a test to see if the lazy creation of Values is actually saving anything

Reply:
_values is only null 4% (5,290,000 vs 226,972 in control catalog ) of the time here so it might be faster to get rid of the lazy initialization and the test. I'll try and benchmark properly.

Reply:
_values is null for 9% of Avalonia objects in Control Catalog so I went for keeping the lazy creation, I was not convinced the small perf advantage was worth the extra memory use.